src/frontend/src/views/Landing.tsx

```diff
@@ -7,7 +7,7 @@ function Landing() {
         fetch(import.meta.env.VITE_API_URL_GREETING_ENDPOINT)
             .then((response) => response.text())
             .then((text) => setData(
-                JSON.parse(text)
```

Comment:
I added this to prevent the text from showing with double quotations. It's not needed?